error/runtime_spec.go

```diff
@@ -41,6 +41,73 @@ const (
 	// ReadonlyOnWindows represents the error code of readonly setting test on Windows
 	ReadonlyOnWindows
 
+	// MountInOrder represents the error code of order test of mount list
+	MountInOrder
+	// MountDestinationAbs represents the error code of absolute path test of mount destination
+	MountDestinationAbs
+	// MountNotNestedOnWindows represents the error code of nested mount point on Windows
+	MountNotNestedOnWindows
+	// MountROSupportOnWindows represents the error code of RO test on Windows
+	MountROSupportOnWindows
+
+	// Process
+	// ProcessOnStart represents the error code of process property test when 'start' is called
+	ProcessOnStart
+	// ProcessIngnoreConsoleSize represents the error code of console size test when terminal is `false` or unset
+	ProcessIgnoreConsoleSize
+	// ProcessCWDAbs represents the error code of absolute path test of process `cwd`
+	ProcessCWDAbs
+
+	// PosixProcess
+	// RlimitType represents the error code of rlimit type test
+	RlimitType
+	// RlimitSoftCur represents the error code of rlimit soft test
+	RlimitSoftCur
+	// RlimitHardMax represents the error code of rlimit hard test
+	RlimitHardMax
+	// RlimitDuplicated represents the error code of rlimit duplicate type test
+	RlimitDuplicated
+
+	// LinuxProcess
+	// LinuxCapabilityMap represents the error code of capability map test
+	LinuxCapabilityMap
+	// LinuxOomScoreAdj represents the error code of `oom_score_adj` value test when it is set
+	LinuxOomScoreAdj
+	// LinuxNoOomScoreAdj represents the error code of `oom_score_adj` value test when it is not set
+	LinuxNoOomScoreAdj
+
+	// Platform-specific configuration
+	// WindowsPlatform represents the error code of windows config test on Windows platform
+	WindowsPlatform
+
+	// POSIX-platform Hooks
+	// PosixHookPathAbs represents the error code of absolute path test of posix hook path
+	PosixHookPathAbs
+	// PosixHookTimeout represents the error code of hook timeout test
+	PosixHookTimeout
```

Comment:
I'd recommend more specific names for future-proofing.  For example, `PosixHookTimeoutPositive` for “[If set, `timeout` MUST be greater than zero][1]”.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#platform-specific-configuration